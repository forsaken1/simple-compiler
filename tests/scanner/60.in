int main() {
	char ch = 'a';
	char *str = "hello world!";
	for(int i = 0; i < 10.45; i++)
		cout << i << endl;

	/* error
	error 
	error */

	return 0; //fuuuu
}