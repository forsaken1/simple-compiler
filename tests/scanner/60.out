1	1	keyword		int
1	5	identificator		main
1	9	separator		(
1	10	separator		)
1	12	separator		{
2	2	keyword		char
2	7	identificator		ch
2	10	operation		=
2	12	char		a
2	15	separator		;
3	2	keyword		char
3	7	operation		*
3	8	identificator		str
3	12	operation		=
3	14	string		hello world!
3	28	separator		;
4	2	keyword		for
4	5	separator		(
4	6	keyword		int
4	10	identificator		i
4	12	operation		=
4	14	integer		0
4	15	separator		;
4	17	identificator		i
4	19	operation		<
4	21	float		10.45
4	26	separator		;
4	28	identificator		i
4	29	operation		++
4	31	separator		)
5	3	identificator		cout
5	8	operation		<<
5	11	identificator		i
5	13	operation		<<
5	16	identificator		endl
5	20	separator		;
11	2	keyword		return
11	9	integer		0
11	10	separator		;
12	1	separator		}
12	2	eof		End of file
